### PR TITLE
Support resolwe test framework

### PR DESCRIPTION
**Pull request by mstajdohar**
Depends on https://github.com/genialis/resolwe/pull/208

### CODE REVIEW

File: resolwe_bio/kb/tests/test_feature.py [hunk 1]
```diff
@@ -42,8 +40,7 @@ def setUp(self):
         for i in range(7, 10):
             self.features.append(FeatureTestCase.create_feature(i, 'XSRC'))
 
-        # Features are not pushed automatically
-        index_builder.push(index=FeatureSearchIndex)
+        self.push_indexes()
```
Comment (dblenkus) [inline resolwe_bio/kb/tests/test_feature.py]: Preserve the comment

File: resolwe_bio/kb/tests/test_feature.py [hunk 2]
```diff
@@ -60,6 +57,7 @@ def assertFeatureEqual(self, data, feature):
         self.assertEqual(data['aliases'], feature.aliases)
 
     def test_feature_search(self):
+
```
Comment (dblenkus) [inline resolwe_bio/kb/tests/test_feature.py]: Delete this line

File: resolwe_bio/kb/tests/test_mapping.py [hunk 3]
```diff
@@ -30,8 +28,7 @@ def setUp(self):
                 target_id='ANOTHER{}'.format(i),
             ))
 
-        # Mappings are not pushed automatically
-        index_builder.push(index=MappingSearchIndex)
+        self.push_indexes()
```
Comment (dblenkus) [inline resolwe_bio/kb/tests/test_mapping.py]: Also here

File: docs/CHANGELOG.rst [hunk 4]
```diff
@@ -6,6 +6,15 @@ All notable changes to this project are documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
 
+==========
+Unreleased
+==========
+
+Added
+-----
+* Support Resolwe test framework
```
Comment (dblenkus) [inline docs/CHANGELOG.rst]: Is this added or changed?

File: tox.ini [hunk 5]
```diff
@@ -16,7 +16,7 @@ commands =
          print(settings.FLOW_EXECUTOR[\"CONTAINER_IMAGE\"])" \
     )\'
 # run tests
-    coverage run --source=resolwe_bio tests/manage.py test --noinput -v 2 {env:TEST_SUITE:resolwe_bio}
+    coverage run tests/manage.py test {env:TEST_SUITE:resolwe_bio} --noinput --verbosity=2
```
Comment (tjanez) [inline tox.ini]: @mstajdohar, did you intentionally omit `--parellel` in this command?
If yes, don't forget to also include `coverage combine` as the next command as [in Resolwe](https://github.com/genialis/resolwe/blob/987328440128926ee4daefa0b0d55e6af35b398d/tox.ini#L13-L15).

File: setup.py [hunk 6]
```diff
@@ -43,10 +43,13 @@
         ]
     },
     zip_safe=False,
+    dependency_links=(
+        "git+https://github.com/genialis/resolwe.git@master#egg=resolwe-master",
+    ),
     install_requires=(
         'Django~=1.10.5',
         'djangorestframework>=3.4.0',
-        'resolwe~=1.4.1',
```
Comment (tjanez) [inline setup.py]: I recently discovered that using `resolwe==master` brakes some Python tools. Please, use `resolwe==1.4.1` and update the dependency link to:
`"git+https://github.com/genialis/resolwe.git@master#egg=resolwe-1.4.1"`.

File: docs/contributing.rst [hunk 7]
```diff
@@ -92,7 +92,7 @@ Run docker::
 
 To run the tests, use::
 
-    ./manage.py test resolwe_bio
+    ./manage.py test resolwe_bio --parallel=2
```
Comment (tjanez) [inline docs/contributing.rst]: Either also enable parallel tests in Tox or omit this change (and make it in a separate commit that enables parallel tests).

File: docs/CHANGELOG.rst [hunk 8]
```diff
@@ -14,6 +14,10 @@ Fixed
 -----
 * Use new import paths in ``resolwe.flow``
 
+Changed
+-------
+* Support Resolwe test framework
```
Comment (tjanez) [inline docs/CHANGELOG.rst]: If you'll enable parallel tests, also note this in the Change log.

File: setup.py [hunk 9]
```diff
@@ -68,6 +71,7 @@
             'pydocstyle>=1.0.0',
             'pylint>=1.6.4',
             'readme_renderer',
+            'tblib>=1.3.0',
```
Comment (tjanez) [inline setup.py]: Why is this necessary?
